dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java

```diff
@@ -391,7 +391,12 @@ private Constants() {
     public static final int RPC_PORT = 50051;
 
     /**
-     * forbid running task
+     * task can be run normally
+     */
+    public static final String FLOWNODE_RUN_FLAG_NORMAL = "NORMAL";
```

Comment:
Hi,
Good job, but there is no necessary to add the `FLOWNODE_RUN_FLAG_NORMAL` in Constants just for test case.
If you want to add the constant, you can directly add `FLOWNODE_RUN_FLAG_NORMAL` in `ConditionsTaskTest`

Reply:
I just moved it to the test file by now, but I think it should be a constant for `"NORMAL"` in `common/Constants.java` for use in future.